src/main/java/org/onedatashare/server/service/ResourceServiceImpl.java

```diff
@@ -113,7 +113,8 @@ public Session createSession(String uri, Credential credential) {
             return new GoogleDriveSession(URI.create(uri), credential);
         }else if(credential instanceof GlobusWebClientCredential){
             return new GridftpSession(URI.create(uri), credential);
-        }
+        } else if(uri.startsWith("box:///"))
```

Comment:
Please replace constant literal with ODSConstants reference

Reply:
Alright, changed that one as well